src/Bicep.LangServer/BicepCompilationManager.cs

```diff
@@ -220,12 +220,16 @@ public void HandleFileChanges(IEnumerable<FileEvent> fileEvents)
                 .Where(pair => pair.Value is CompilationContext)
                 .Select(pair => new KeyValuePair<DocumentUri, CompilationContext>(pair.Key, (CompilationContext)pair.Value));
 
-        private bool ShouldUpsertCompilation(DocumentUri documentUri, string? languageId = null)
+        private bool ShouldUpsertCompilation(DocumentUri documentUri, string? languageId = null) //asdfg
```

Comment:
```suggestion
        private bool ShouldUpsertCompilation(DocumentUri documentUri, string? languageId = null)
```

Assuming this was a placeholder; did you mean to put different comments here and below?

Reply:
Sorry, already fixed.